fastlane/Fastfile

```diff
@@ -37,14 +37,16 @@ APP_STORE_VERSION_BUNDLE_IDENTIFIER = 'com.automattic.woocommerce'
 # Registered in our main account, for development and App Store
 MAIN_BUNDLE_IDENTIFIERS = [
   APP_STORE_VERSION_BUNDLE_IDENTIFIER,
-  "#{APP_STORE_VERSION_BUNDLE_IDENTIFIER}.storewidgets"
+  "#{APP_STORE_VERSION_BUNDLE_IDENTIFIER}.storewidgets",
+  "#{APP_STORE_VERSION_BUNDLE_IDENTIFIER}.notificationcontentextension"
 ].freeze
 
 ALPHA_VERSION_BUNDLE_IDENTIFIER = 'com.automattic.alpha.woocommerce'
 # Registered in our Enterprise account, for App Center / Prototype Builds
 ALPHA_BUNDLE_IDENTIFIERS = [
   ALPHA_VERSION_BUNDLE_IDENTIFIER,
-  "#{ALPHA_VERSION_BUNDLE_IDENTIFIER}.storewidgets"
+  "#{ALPHA_VERSION_BUNDLE_IDENTIFIER}.storewidgets",
+  "#{ALPHA_VERSION_BUNDLE_IDENTIFIER}.notificationcontentextension"
```

Comment:
Next time we add an extension, we might want to refactor this to DRY. 

Eg. WordPress [injects the bundle identifier root](https://github.com/wordpress-mobile/WordPress-iOS/blob/trunk/fastlane/Fastfile#L33-L40) and computes the Enterprise ones [via substitution](https://github.com/wordpress-mobile/WordPress-iOS/blob/1cc0fc7c0d75cdc945296d6afc79bf65d016870a/fastlane/lanes/codesign.rb#L105).

Reply:
Good to know! I'd also like to DRY the DOiOS/Mac ones, so I'll make a note to update WCiOS as well.